packages/forms/src/UIForm/UIForm.component.test.js

```diff
@@ -252,6 +253,11 @@ describe('UIForm component', () => {
 	});
 
 	describe('#onSubmit', () => {
+		const validProperties = {
```

Comment:
This seems to be used in 1 test, please move that back in the test

Reply:
done